packages/webapp/src/containers/Filter/Transactions/index.jsx

```diff
@@ -0,0 +1,86 @@
+/*
+ *  Copyright 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+import { EXPENSE_TYPE, REVENUE_TYPE } from '../constants';
+import { expenseTypeSelector } from '../../Finances/selectors';
```

Comment:
I think we need to include retired types (`allExpenseTypeSelector`, `allRevenueTypeSelector`) as well, based on the fact that the retired modal mentions filtering. But then I wonder if there should be a visual difference between retired and active types on the pills?

<img width="538" alt="Screenshot 2023-11-02 at 6 04 54 PM" src="https://github.com/LiteFarmOrg/LiteFarm/assets/82857964/60cf0aad-68bc-45f9-9db6-05b547f52392">

Reply:
Thank you for pointing this out! I didn't have any retired types so I didn't notice it

Reply:
Btw I checked with Loïc and he said we don't need a visual distinction for retired types in the pills

Reply:
This is a reach goal -- if it gets implemented I think a new ticket is appropriate -- but do you think someday we should check for whether a retired type is associated with any transactions before showing it in the filters?